RecoTracker/IterativeTracking/python/MixedTripletStep_cff.py

```diff
@@ -344,13 +345,14 @@
     qualityCuts = [-0.5,0.0,0.5]
 ))
 
-from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
-from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-trackdnn.toReplaceWith(mixedTripletStep, TrackLwtnnClassifier.clone(
-    src = 'mixedTripletStepTracks',
-    qualityCuts = [-0.8, -0.35, 0.1]
+from RecoTracker.FinalTrackSelectors.TrackTfClassifier_cfi import *
+from RecoTracker.FinalTrackSelectors.trackSelectionTf_cfi import *
+trackdnn.toReplaceWith(mixedTripletStep, TrackTfClassifier.clone(
+     src = 'mixedTripletStepTracks',
+     qualityCuts = qualityCutDictionary['MixedTripletStep']
```

Comment:
you should remove the extra whitespace that was added, python is picky about exactly 4 spaces 
```suggestion
    src = 'mixedTripletStepTracks',
    qualityCuts = qualityCutDictionary['MixedTripletStep']
```